### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by johnkioko**
In this project, I have added some new tables and added foreign key columns to your existing animal's table to model one-to-many relationships. Afterward, I use JOIN to query the data.

query_join_1
query_join_2

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -13,4 +13,28 @@ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALU
...
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodie Whittake', 38);
+
+INSERT INTO species (name) VALUES ('Pokemon'),('Digimon');
+
+/*Modify your inserted animals so it includes the species_id value*/
+
+UPDATE animals SET species_id = 1 WHERE name LIKE '%mon';
+UPDATE animals SET species_id = 2 WHERE name NOT LIKE '%mon';
```
Comment (Deepakdanger) [inline data.sql]: - [ ] Write the correct update.
```
UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Pokemon')
   WHERE species_id IS NULL;
```

File: queries.sql [hunk 2]
```diff
@@ -48,4 +48,37 @@ SELECT COUNT(*) FROM animals WHERE escape_attempts < 1;
...
 SELECT neutered, MAX(escape_attempts) FROM animals GROUP BY neutered;
 SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
-SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
\ No newline at end of file
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+
+/* Day 3 */
+
+SELECT name FROM animals
```
Comment (Deepakdanger) [inline queries.sql]: - [ ] we have to select all amimals , Kindly use `*`, ``SELECT * FROM animals``.

File: queries.sql [hunk 3]
```diff
@@ -48,4 +48,37 @@ SELECT COUNT(*) FROM animals WHERE escape_attempts < 1;
...
+JOIN owners ON owners.id = animals.owner_id
+WHERE owners.full_name = 'Melody Pond';
+
+SELECT animals.name FROM animals
```
Comment (Deepakdanger) [inline queries.sql]: - [ ] we have to select all amimals , Kindly use `*`, ``SELECT * FROM animals``

**Review by Deepakdanger — APPROVED** [review]
Hi @johnkioko , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Deepakdanger** [comment]
Kindly implement changes & ask for next review

**Review by EricMbouwe — APPROVED** [review]
Hi @johnkioko, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Requested changes ✔
- PR ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**